### PR TITLE
Eliminate TrPred from frr family

### PR DESCRIPTION
**Pull request by sctfn**
Continuation of #4454 . This eliminates all usage of TrPred outside of its own section. Do we have a preferred way to obsolete an entire section?

### CODE REVIEW

**Review by tirix — APPROVED** [review]

File: set.mm [hunk 1]
```diff
@@ -101409,8 +101409,26 @@ a Cantor normal form (and injectivity, together with coherence
   ${
     $d R w z $.  $d A w z $.
     $( Lemma for general well-founded recursion.  Establish a subset
-       relationship.  (Contributed by Scott Fenton, 11-Sep-2023.) $)
+       relationship.  (Contributed by Scott Fenton, 11-Sep-2023.)  (Revised by
```
Comment (wlammen) [inline set.mm:101412]: Note the reason for this revision (Transitivity revisited, for example)

**Review by wlammen — CHANGES_REQUESTED** [review]

**Comment by icecream17** [comment]
Obsoleting an entire section: Probably add discouraged tags or move the section to your mathbox (I've seen both)

File: set.mm (hunk 1 above)
Comment (sctfn) [reply set.mm]: Done

**Comment by sctfn** [comment]
I just went ahead and eliminated TrPred since it's completely subsumed by t++

**Review by wlammen — APPROVED** [review]

**Pull request merged**